UnitTestBot/UTBotJava
# Fix test framework loading from file

## Description

Changed loading of the test framework settings from config file: 
1. We parse text representation of the saved test framework
2. If it is not for java (Junit5, Junit4, Testng) we interpret as `UnknownTestFramework`
3. Then in Python and in Java UI logic we transform `UnknownTestFramework` to real test framework throw `JavaTestFrameworkMapper` or `PythonTestFrameworkMapper`

Fixes #1563

## How to test

### Manual tests

See #1563 and:

1. Open java project in IntelliJ IDEA
2. Check that test framework in UTBot UI is Junit4/5/Testng
3. Switch off `experimental languages` in UTBot settings
4. Close IDE and reopen python project and switch on `experimental languages support`
5. Try to generate tests with `pytest` (__expected__: tests have been created)
6. Close IDE and reopen python project
7. Check that `experimental languages` is active and default test framework is `pytest`
8. Close IDE again and reopen java project
9. Check that `experimental language` is active and there are no problems with java test framework

## Self-check list

Check off the item if the statement is true. Hint: [x] is a marked item.

Please do not delete the list or its items.
